<launch>

	<arg name="namespace" default="Bilbo" />
    <node pkg="yolo_triangulation" type="yolo_triangulation" name="triangulation" output="screen">

		<!-- Camera parameters -->
		<param name="fx" value="609.9031982421875"/>
		<param name="fy" value="609.9591064453125"/>
		<param name="cx" value="320.2873229980469"/>
		<param name="cy" value="239.86740112304688"/>

		<!-- Camera position in body frame (in meters) -->
		<rosparam param="cam_pos_body_frame"> [0.15, 0.0, 0.0] </rosparam>

		<!-- Subscribing topics -->
		<param name="input_pose_topic" value="/$(arg namespace)/local_odom"/>
		<param name="input_bounding_boxes_topic" value="/darknet_ros/bounding_boxes"/>
    
	</node>

</launch>